server/routes/dashboard/dashboard.ts

```diff
@@ -74,6 +67,15 @@ export default function dashboard(service: Services): Router {
       order = 'DESC'
     }
 
+    if (
+      userRoles.includes(roleReadWrite) &&
+      !userRoles.includes(roleApproveReject) &&
+      !incidentStatuses &&
+      req.url !== '/?'
```

Comment:
what is `req.url !== '/?'` checking for?

Reply:
oh i see, it's when u clear filters. i would imagine that `/` & `/?` should show the same thing and that clearing filters would actually reset them to the default, ie. with to-do checked. don't u think?

Reply:
in which case you could check for presence or absence of `'incidentStatuses' in req.query` to allow for a deliberately empty selection of checkboxes (which isn't the default)